tools/shared.py

```diff
@@ -931,8 +935,10 @@ def get_llvm_target():
 
 if LLVM_TARGET == WASM_TARGET:
   # wasm target does not automatically define emscripten stuff, so do it here.
-  COMPILER_OPTS = COMPILER_OPTS + ['-DEMSCRIPTEN',
-                                   '-D__EMSCRIPTEN__']
+  COMPILER_OPTS = COMPILER_OPTS + ['-D__EMSCRIPTEN__']
```

Comment:
I'm not sure if adding that is much worth it, since this occurs exactly in two places, and it's quite lean to just add it in these locations. Eventually we'll be tearing down the code anyways when we get to remove `#define EMSCRIPTEN` altogether, so building a new abstraction for this seems temporary? Left this one as is for now, let me know if you feel strongly about this.